### PR TITLE
docs: document applyDefaultOnWrites and persistDefaultValues

### PR DESCRIPTION
**Pull request by hacksparrow**
Document applyDefaultOnWrites and persistDefaultValues

### CODE REVIEW

File: pages/en/lb3/Model-definition-JSON-file.md [hunk 1]
```diff
@@ -341,6 +341,22 @@ For example:
...
+      <td>
+        When set to `false`, does not write the default value to the database if the property value is missing. Defaults to `true`.
+      </td>
+    </tr>
+    <tr>
+      <td>persistDefaultValues</td>
+      <td>No</td>
+      <td>Boolean</td>
+      <td>
+        When set to `false`, does not write the default value to the database if the property value matches the default value. Defaults to `true`. Applicable only for primitive data types.
```
Comment (agnes512) [inline pages/en/lb3/Model-definition-JSON-file.md:357]: `Applicable only to` ?

**Review by agnes512 — APPROVED** [review]
LGTM, only a comment 👍

File: pages/en/lb3/Model-definition-JSON-file.md (hunk 1 above)
Comment (hacksparrow) [reply pages/en/lb3/Model-definition-JSON-file.md:357]: Either is fine.

**Review by jannyHou — COMMENTED** [review]
@hacksparrow 👍 The content LGTM, you may also want to add them for LB4:
https://loopback.io/doc/en/lb4/Model.html#supported-entries-of-settings
or
https://loopback.io/doc/en/lb4/Model.html#unsupported-entries-of-settings

**Review by jannyHou — APPROVED** [review]

**Comment by hacksparrow** [comment]
@jannyHou these are properties of model properties, not model properties.

**Comment by agnes512** [comment]
Btw is this also applicable to LB4?

**Comment by hacksparrow** [comment]
Yes.

**Pull request merged**